examples/bull-spread/bull-spread.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"cloud.google.com/go/civil"
+	"github.com/shopspring/decimal"
+
+	"github.com/alpacahq/alpaca-trade-api-go/v3/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/v3/marketdata"
+)
+
+var decimal10k = decimal.NewFromInt(10000)
```

Comment:
[minor] Please move this before the line it's used (currently L53)

Reply:
a0545df8bfa0e00b8922f8b06424d92b6fc7995a